src/ranges.js

```diff
@@ -113,9 +113,100 @@ const findSoleUncommonTimeRangesEnd = function(original, update) {
   return result[0];
 };
 
+/**
+ * Calculate the intersection of two TimeRanges
+ * @param bufferA {TimeRanges}
```

Comment:
I want to blame someone else for these broken style comments. It wasn't me, I don't even write comments! :wink: